.github/workflows/release.yaml

```diff
@@ -340,94 +371,3 @@ jobs:
         with:
           files: |
             release/*
-
-  build-aws-image:
-    runs-on: ubuntu-latest
-    needs:
-      - get-standard-matrix
-      - build-standard
-    permissions:
-      id-token: write  # OIDC support
-      contents: write
-      actions: read
-      security-events: write
-    strategy:
-      matrix:
-        include:
-          # We don't publish AWS images for all combinations so we go hardcoded here
-          - flavor: ubuntu
-            flavor_release: 24.04
-            family: ubuntu
-            base_image: ubuntu:24.04
-            variant: standard
-    steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
-      - run: |
-          git fetch --prune --unshallow
-      - name: Install kairos-agent (for versioneer)
-        uses: Luet-lab/luet-install-action@cec77490c3f2416d7d07a47cfab04d448641d7ce # v1.1
-        with:
-          repository: quay.io/kairos/packages
-          packages: system/kairos-agent
-      - name: Build 🔧
-        run: |
-          # Keep maximum k3s version (by semver)
-          # add a "v" in front and a "-k3s1" at the end to match the k3s versioning
-          k3s_version=$(echo '${{ needs.get-standard-matrix.outputs.matrix }}' | \
-            jq -r '.include | map(.k3s_version | split("+")[0]) | unique |
-            map(
-              capture("^(?<major>[0-9]+)\\.(?<minor>[0-9]+)\\.(?<patch>[0-9]+)(-(?<pre>.*))?$") |
-              {
-                major: (.major | tonumber),
-                minor: (.minor | tonumber),
-                patch: (.patch | tonumber),
-                is_stable: (if .pre == null then 1 else 0 end),
-                pre: (.pre // "")
-              }
-            ) | max_by(.major, .minor, .patch, .is_stable, .pre) |
-              "\(.major).\(.minor).\(.patch)\(.pre | if . == "" then "" else "-"+. end)"' | \
-            sed -E 's/^/v/; s/$/-k3s1/')
```

Comment:
This "worked" kind of, but the artifacts we push are named:

```
 quay.io/kairos/ubuntu:24.10-standard-amd64-generic-v3.3.1-rc2-k3sv1.32.1-rc2-k3sk3s1
```
and this script was looking for:

```
quay.io/kairos/ubuntu:24.04-standard-amd64-generic-v3.3.1-rc2-k3sv1.32.1-rc2-k3s1
```

I think the first one is wrong which is a different issue to look at but in any  case, I thought it would be better if we didn't try to construct the image name again so this PR tries to push the AWS at a point where the image name is still available.

Reply:
https://github.com/kairos-io/kairos/pull/3156